src/bp/lang-server/service/index.ts

```diff
@@ -3,22 +3,43 @@ import fs from 'fs'
 import _ from 'lodash'
 import lru from 'lru-cache'
 import ms from 'ms'
+import os from 'os'
 import path from 'path'
+import process from 'process'
 import { VError } from 'verror'
-
 import toolkit from '../../ml/toolkit'
 import Logger from '../../simple-logger'
-
 import { LoadedBPEModel, LoadedFastTextModel, ModelFileInfo, ModelSet } from './typing'
 
+interface RamInfos {
+  free: number
+  total: number
+  prediction: number
+}
+
 const maxAgeCacheInMS = ms('24h')
 
+const MARGIN_PRED = 0.2 // Adds some % to the predicted ram usage
```

Comment:
can you name this properly and remove the comment.
`MODEL_SAFETY_BUFFER` or something like this

Reply:
Done and did the same bellow for the ram threshold